packages/schematics/angular/app-shell/index.ts

```diff
@@ -171,12 +170,16 @@ function addUniversalTarget(options: AppShellOptions): Rule {
     // Copy options.
     const universalOptions = {
       ...options,
-      name: options.universalProject,
     };
 
     // Delete non-universal options.
     delete universalOptions.universalProject;
     delete universalOptions.route;
+    delete universalOptions.name;
```

Comment:
This is a recurring pattern. Have you thought about adding a `pluck` or `filterKeys` function instead?

Reply:
At this point no, but I think getting this merged is more important than that refactor.